deepchem/models/tensorgraph/layers.py

```diff
@@ -1177,18 +1214,19 @@ def __init__(self, shapes, names, capacity=5, **kwargs):
     super(InputFifoQueue, self).__init__(**kwargs)
 
   def create_tensor(self, in_layers=None, **kwargs):
-    # TODO(rbharath): Note sure if this layer can be called with __call__
-    # meaningfully, so not going to support that functionality for now.
     if in_layers is None:
       in_layers = self.in_layers
     in_layers = convert_to_layers(in_layers)
     self.dtypes = [x.out_tensor.dtype for x in in_layers]
-    self.queue = tf.FIFOQueue(
-        self.capacity, self.dtypes, shapes=self.shapes, names=self.names)
+    self.queue = tf.contrib.staging.StagingArea(
```

Comment:
@peastman this is the important part. Highlighting it for you. Note that tf1.2 vs tf1.3 apis are slightly different.